### ISSUE TITLE
JAWS doesn't announce aria-selected state of gridcell in forms mode but does in virtual pc cursor mode

### ISSUE THREAD

**Issue by Jan-Michael-Koensgen-SAP** [FreedomScientific/VFO-standards-support#615]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Open [grid](https://codepen.io/JAN-MICHAEL-KOENSGEN-SAP/pen/YzYYoPZ) in chrome
> 2. Start JAWS
> 3. Click "cell 1" or "cell 2" or "cell 3"

### Expected result

> The selection state of the cell gets announced in JAWS forms mode

### Actual result

> The selection state of each cell set by aria-selected="true" does not get announced in JAWS forms mode but does get announced in JAWS virtual pc cursor mode

### Example

> [CodePen](https://codepen.io/JAN-MICHAEL-KOENSGEN-SAP/pen/YzYYoPZ)

#### JAWS version and build number
2021.2111.13

**Comment by ling1726** [comment]
I can repro this too
